api-tester/src/defaults/kotlin/com/revenuecat/apitester/kotlin/PurchasesAPI.kt

```diff
@@ -105,6 +110,11 @@ private class PurchasesAPI {
         val customerInfo: CustomerInfo = purchases.awaitCustomerInfo()
         val customerInfoFetchPolicy: CustomerInfo =
             purchases.awaitCustomerInfo(fetchPolicy = CacheFetchPolicy.FETCH_CURRENT)
+        val logInResult: LoginResult = purchases.awaitLogIn("appUserID")
+        val (customerInfo2: CustomerInfo, created: Boolean) = purchases.awaitLogIn("appUserID")
```

Comment:
I guess this might kinda cover API tests for `LogInResult`, but maybe we should have them explicitly still?